style.css

```
@@ -0,0 +1,483 @@
+    padding: 10px; 
+}
+.progress-bar:before {
+    content: "";
+    position: absolute;
+    right: 0;
+    width: 50%;
+    z-index: 0;
+}
+/*===== The CSS =====*/
+.progress{
+    width: 200px;
+    height: 280px;
+    z-index: 1;
+}
+.progress .track, .progress .fill{
+    fill: rgba(0, 0, 0, 0);
+    stroke-width: 3;
+    transform: rotate(90deg)translate(0px, -80px);
+}
+.progress .track{
+    stroke: #191923;
+}
+.progress .fill {
+    stroke: #0ce353;
+    stroke-dasharray: 219.99078369140625;
+    stroke-dashoffset: -219.99078369140625;
+    transition: stroke-dashoffset 1s;
+}
+.progress .value {
+    fill: #0ce353;
+    text-anchor: middle;
+}
+.progress .text {
+    font-size: 12px;
+    fill: #fff;
+    text-anchor: middle;
+}
+
+#skills-block {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    font-size: 14px;
+    padding: 20px 30px;
+    margin-bottom: 10px;
+}
+
+.barra-progresso-pai {
+    margin-top: 10px;
+    width: 100%;
+    box-sizing: border-box;
+}
+
+#skills-block p {
+    color: white;
+    font-size: 12px;
+}
+
+#skills-block span {
+    float: right;
+}
+
+#skills-tools-block {
+    padding: 20px 30px;
+}
+
+#skills-tools-block p {
+    color: #8c8c8e;
+    font-size: 14px;
+    line-height: 25px;
+}
+
+.download-cv-btn {
+    background-color: transparent;
+    border: none;
+    color: #8c8c8e;
+    font-size: 12px;
+    font-weight: 600;
+    letter-spacing: 2px;
+    padding: 20px 30px;
+    
+}
+
+.download-cv-btn a {
+    text-decoration: none;
+    color: #8c8c8e;
+}
+
+#scrollable-content {
+    background-color: #20202A;
+    height: 60%;
+    max-height: 100vh;
+    overflow-y: auto;
+    scrollbar-width: thin;
+}
+
+#scrollable-content::-webkit-scrollbar {
+    width: 7px;
+}
+
+#scrollable-content::-webkit-scrollbar-thumb {
+    background-color: #0ce353;
+    border-radius: 4px;
+}
+
+#social-block {
+    align-items: center;
+    background-color: #24242F;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    height: 5%;
+    width: 300px;
+    
+}
+
+#social-block i {
+    color: #8c8c8e;
+    margin: 0 10px;
+}
+
+#menu-block {
+    box-shadow: 1px 1px 1px 3px rgba(0, 0, 0, 0.1);
+    height: 100vh;
+    right: 0;
+    position: fixed;
+    width: 100px;
+}
+
+.menu-btn {
+    background-color: transparent;
+    border: none;
+}
+
+.menu-btn-block {
+    background-color: #24242F;
+    display: flex;
+    height: 100px;
+    justify-content: center;
+}
+
+#main-block {
+    position: relative;
+    height: 100%;
+    margin-left: 300px; 
+    margin-right: 100px;
+    overflow-y: auto;
+    scrollbar-width: thin;
+}
+
+#main-block::-webkit-scrollbar {
+    width: 7px;
+}
+
+#main-block::-webkit-scrollbar-thumb {
+    background-color: #0ce353;
+    border-radius: 4px;
+}
+
+#main-block::before {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 50%;
+    background-image: url("./images/215911.gif");
```

de um nome melhor para o gif.

```suggestion
    background-image: url("./images/pc-vibe.gif");
```

---

Feito, thanksssssss